app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script/api_create.rb

```diff
@@ -2,21 +2,23 @@ class ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript
   module ApiCreate
     def create_in_provider(manager_id, params)
       manager = ExtManagementSystem.find(manager_id)
+      raise "authentication status is not ok" unless manager.authentication_status_ok?
```

Comment:
This is discovered through test. You can reproduce and try a better solution. I have a valid tower but the `authentication_status_ok?` returns false because the appliance (dev machine) was disconnected from VPN before. The job template can be created successfully because it does not check for authentication status, but refresh cannot pass through. The created job template is useless because we cannot get it in VMDB.

We can attempt to sync the current authentication status. This line can be a temporary solution before we implement a better way.